internal/exec/util/unit.go

```diff
@@ -110,8 +113,58 @@ func (ut Util) MaskUnit(unit types.Unit) (string, error) {
 	return filepath.Join("/", SystemdUnitsPath(), unit.Name), nil
 }
 
+var builder = strings.Builder{}
+
+// isRequiredSystemdVersionExists checks if the version of systemd is greater
+// than or equal to 240 for leveraging this (https://github.com/systemd/systemd/pull/9901) feature merged in the v240 branch.
+func isRequiredSystemdVersionExists() (bool, error) {
+	out, err := exec.Command("chroot", "/sysroot", "rpm", "-q", "systemd").Output()
```

Comment:
since ignition theoretically supports non rpm based distros let's use `systemctl --version` here. That should also mean we don't need to `chroot` at all.

Reply:
FYI you can get directly query this property from systemd dbus (e.g. via go-systemd).